README.adoc

```diff
@@ -297,6 +297,20 @@ in_cluster: (true\|false)
 istio_sidecar_annotation: VALUE
 ----
 
+|`ISTIO_SERVICE_NAMESPACE`
+|The Kubernetes namespace that holds the Istio service
+[source,yaml]
+----
+istio_service_namespace: VALUE
```

Comment:
Can you show how this property is now nested:
```
products:
   istio:
      istio_service_namespace: VALUE
```
and the same for other now nested properties?

Reply:
Yes, I forgot update the readme , thanks @jotak

Reply:
I added some configuration that was missed and changed @jotak